src/wallet/signer.rs

```diff
@@ -427,17 +532,47 @@ impl SignersContainer {
     }
 }
 
+/// Options for a software signer
+///
+/// Adjust the behavior of our software signers, [`ExtendedKeySigner`] and [`PrivateKeySigner`]
+#[derive(Debug, Clone)]
```

Comment:
Maybe we can add `Copy` since the struct content is small, will allow removing some explicit call to `clone` around

Reply:
now the struct has more than one field